openvino_node/launch/object_detection.launch.py

```diff
@@ -4,7 +4,7 @@
 from ament_index_python.packages import get_package_share_directory
 
 def generate_launch_description():
-    print(get_package_share_directory('openvino_ros')+'/config/object_detection.yaml')
+    print(get_package_share_directory('openvino_node')+'/config/object_detection.yaml')
```

Comment:
This line is for debugging. I forget to remove it. Please delete it if possible.

Reply:
As talked, we'd like to keep this line to help user quick locate the config file